app/routes/configurations/overview.tsx

```diff
@@ -59,11 +78,15 @@ const ConfigCard = ({ config }: { config: ConfigurationIdentifier }) => {
   );
 };
 
-const useConfigTags = (config: ConfigurationIdentifier) => {
+const useTags = () => {
   const { t } = useTranslation();
-  const tags = [];
-  if (config.project.isIar) {
-    tags.push(t('common.label.readOnly'));
-  }
-  return tags;
+  const allTags: Array<string> = [t('common.label.readOnly')];
+  const tagsFor = (config: ConfigurationIdentifier) => {
+    const tags: Array<{ label: string; tagStyle: TagStyle }> = [];
```

Comment:
I would just export a whole Tag type in the ArtifactTag file which contains `{ label: string; tagStyle: TagStyle }` and you can use everywhere


```suggestion
    const tags: Array<Tag> = [];
```